plugin-server/src/main/ingestion-queues/session-recordings-consumer.ts

```diff
@@ -353,6 +348,7 @@ const createKafkaProducer = async (config: ProducerGlobalConfig) => {
         // but at least larger than the default.
         'batch.size': 1024 * 1024, // bytes. The default
         'compression.codec': 'snappy',
+        'enable.idempotence': true,
```

Comment:
nit: I think it's worth commenting why we enable it and what related assumptions we now depend on. We might be tempted to disable it later on.

Reply:
Have added a comment.